ignite-base/App/Services/Api.js

```diff
@@ -21,6 +21,11 @@ const create = (baseURL = 'http://openweathermap.org/data/2.1') => {
     timeout: 10000
   })
 
+  // Force OpenWeather API Key on all requests
+  api.addRequestTransform(request => {
+    request.params['APPID'] = '0e44183e8d1018fc92eb3307d885379c'
```

Comment:
It's a unicorn.  A rare mythological beast.  Legend says that its presence alone is enough to ward off api key thieves.